parser/parse_test.go

```diff
@@ -159,8 +161,15 @@ func TestBuilder(t *testing.T) {
 
 				type Enumeration string
 				const (
-					EnumSymbol Enumeration = "enumSymbol"
+					EnumSymbol Enumeration = "enumSymbolValue"
 				)
+
+				type Bytes int
```

Comment:
I know it's test code, but "Bytes int" is confusing :)

Reply:
Renamed to "Degrees"